src/App.js

```diff
@@ -1,9 +1,11 @@
 import './App.css';
+import { Navigation } from './components/Navigation/Navigation';
```

Comment:
With your current approach of importing if you have to import 10 components, it will take 10 lines
Instead, keep an `index.js` file inside the components folder and import and export all components from there.
In this way, you'll have all your imports in one file and from there they can easily be imported into other files

Reply:
Sure. Thanks for the suggestion.